dashboard/test/controllers/api_controller_test.rb

```diff
@@ -1706,6 +1706,45 @@ def get_student_response(script, level, lesson, student_number)
     assert_response :forbidden
   end
 
+  test 'clever section name too long' do
+    teacher = create :teacher, :with_clever_authentication_option
+    sign_in teacher
+    course_name = 'test' * 65
+
+    ApiController.any_instance.expects(:query_clever_service).yields(clever_service_response_stub)
```

Comment:
Since the changes no longer affect our handling of the API response, the tests could be cleaned up by moving them into the omni auth section tests. Saves the trouble of having to stub the API response - you could just test the long input to the from_omniauth method.